src/components/PeopleTableBody/PeopleTableBody.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
```

Comment:
Similar names are okay for a small project, but on large projects we will have many interfaces and types, so it is better to give more detailed names, for example `PeopleTableBodyProps`.
This applies to all code.